src/bootstrap/bootstrap.ts

```diff
@@ -4,103 +4,161 @@ import path from 'path';
 import fs from 'fs';
```

Comment:
This one needs to be replaced with DI.

But this whole file should be `src/bootstrap/utils.ts`.